charter.md

```diff
@@ -0,0 +1,32 @@
+# Style Team
+
+## Aims: Evolving the Rust Style over time
+
+1. Selecting styling for new Rust constructs
```

Comment:
```suggestion
1. Defining the style for new Rust constructs
```

("selecting" suggests we can only pick from options, rather than creating options)